### ISSUE TITLE
[Bug]: Side Bar does not show HP nor AC

### ISSUE THREAD

**Issue by namtrah** [javalent/initiative-tracker#162]
SYSTEM INFO:
	Obsidian version: v1.2.6
	Installer version: v1.1.16
	Operating system: #39~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Mar 17 21:16:15 UTC 2 5.19.0-38-generic
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: ITS Theme v0.8.87
	Snippets enabled: 36
	Restricted mode: off
	Plugins installed: 61
	Plugins enabled: 57
		2: Initiative Tracker v9.8.2

In the Initiative Tracker Side Bar after upgrading to v9.8.2, the HP and AC are no longer seen.
https://cdn.discordapp.com/attachments/932707309195493416/1102158880605220946/image.png
## What is the expected behavior?
Previously after an upgrade you could still see the HP and AC as set in each Player's MD File (using the YAML `ac:`  `hp:`.) I presume Level and Modifier are the same.

## What is the current behavior?
As the image shows the values are `-`. When selecting "Reset AC", the AC at least changes back to the real AC, until one tries to affect HP.

## What are the A-to-Z Steps to Reproduce your Bug?
1. Install v9.8.2 (happens before, during and after a full reload and restart of Obsidian)
2. Open IT Side Bar

## Error Codes
[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
configure @ plugin:initiative-tracker:30611
update2 @ plugin:initiative-tracker:30626
update2 @ plugin:initiative-tracker:31097
p @ plugin:initiative-tracker:31293
p @ plugin:initiative-tracker:31465
update @ plugin:initiative-tracker:3191
flush @ plugin:initiative-tracker:3162

## Screenshots and Video
See Discord for Screenshots.

**Comment by sigrunixia** [comment]
![image](https://user-images.githubusercontent.com/68425372/235375301-24fc8c36-bbf4-408f-bba2-93d4f9a1b038.png)

![image](https://user-images.githubusercontent.com/68425372/235375303-b3a9cd0d-5337-4b5f-a04f-8598a07c313d.png)

![image](https://user-images.githubusercontent.com/68425372/235375307-2a918efb-4d6a-45f8-8044-023d5abf3dcc.png)


Added Images from Discord

**Comment by sigrunixia** [comment]
@namtrah 

Do you have images in Mr. Ghost's note?
Does the issue persist when the note is disconnected?

**Comment by namtrah** [comment]
No images (although in my other players, yes), here is the whole content of Mr. Ghost. I am not certain what you mean by "disconnected". How would I do that with IT for one note? 

```markdown
---
aliases: ['Mr. Ghost']
type: pc
tags: 
- npc/alive
- iff/unknown
- location
hp: 10
ac: 10
modifier: 0
level: 4
---
# Mr. Ghost
<span class="subhead">{{primary location}}</span>

TL;DR description / personality / motivation

> *{{gender}} {{race}} {{role/occupation}}, {{alignment}}*  
> * **Trait**
> * **Ideal**
> * **Bond**
> * **Flaw**

<span class="nav">[Details](#Details) [Relationships](#Relationships) [Secrets](#Secrets) [History](#History)</span>

## Details


## Relationships

**Organization or Faction**

## Secrets

## History

Any queries?
```

**Comment by namtrah** [comment]
OK, so I finally found "**Reset HP and Status**" option in the menu and after that, all returned to normal. Even after a **Reload App without Saving** of Obsidian. So it may be in the initialization of IT?
![image](https://user-images.githubusercontent.com/13903735/235430098-20f5835c-6977-4849-9be7-c5961eaf19af.png)

**Comment by sigrunixia** [comment]
Possibly. There was another user who had issues if they had images on top of their note. As there seems to be a resolution here, and we're not getting any other reports, I will close this for now.
